bypass activate flag not set about bsc#1073658

michelmno opened the pull request:
to continue on related error message
assuming existance of new needle "activate_flag_not_set"
https://github.com/os-autoinst/os-autoinst-needles-opensuse/pull/314

Signed-off-by: Michel Normand <normand@linux.vnet.ibm.com>

rwx788 commented on tests/installation/start_install.pm:
```diff
@@ -88,9 +88,10 @@ sub run {
             last if match_has_tag("startinstall");
-            send_key $cmd{acceptlicense};
+            send_key $cmd{acceptlicense} if match_has_tag("confirmlicense");
+            send_key 'alt-o';
```
Now if needle matches 'confirmlicense' tag we will still press alt-o, and I'm not sure if that is required. Should it be pressed when match_has_tag activate_flag_not_set? As this tag is not used anywhere.

michelmno replied:
Oups, patch updated to not press alt-o for license.
I also added above in first comment the ref to needle PR.

rwx788 commented on tests/installation/start_install.pm:
```diff
@@ -88,9 +88,13 @@ sub run {
+            if match_has_tag("confirmlicense") {
+                send_key $cmd{acceptlicense};
+            } else { 
+                send_key 'alt-o';
```
As it's a workaround, could you please add record_soft_failure with a bug reference? I'm also not sure if this will help much and SUT won't be able to boot...

michelmno replied:
The needle suffix with bug number is sufficient to identify the bypass.
A test in my local openQA instance confirmed that boot completed.

michelmno commented:
@rwx788 I updated the patch and added ref to needle PR in first comment.
I am pushing this PR because correction of related bug did not yet meet openQA PowerPC and need more investigation for other problems (no snapshot after 20180124) https://openqa.opensuse.org/group_overview/4
